src/tvdb/client.py

```diff
@@ -212,12 +215,31 @@ class Series(_Media):
     ResponseType = SeriesIdGetResponse
     ExtendedResponseType = SeriesIdExtendedGetResponse
 
+    def __init__(self, client: "TvdbClient", data: AnyRecord | SearchResult | None):
+        super().__init__(client, data)
+        self.episodes: list[EpisodeBaseRecord] | None = None
+        self.seasons: list[SeasonBaseRecord] | None = None
+        if isinstance(self.data, SeriesExtendedRecord):
+            self.seasons = self.data.seasons
+
     @override
     @classmethod
     async def supports_meta(cls, meta: FetchMeta) -> bool:
         """Check if the class supports a specific meta."""
         return meta in {FetchMeta.TRANSLATIONS, FetchMeta.EPISODES}
 
+    async def find_episodes(self, session: aiohttp.ClientSession, season_type: str = "official") -> None:
```

Comment:
```suggestion
    async def fetch_episodes(self, session: aiohttp.ClientSession, season_type: str = "official") -> None:
```

Since this isn't returning anything, I find `fetch` a bit better word to use, as it just populates an internal variable with something fetched from the API.